legwork/psd.py

```diff
@@ -143,14 +154,23 @@ def Sc(f, t_obs):
     else:
         R = load_response_function(f, fstar)
 
+    # if sky position and polarization are specified, include them
+    # instead of the averaged response; to do this, we need to undo
+    # the 10/3 averaging term
+    if (theta is not None) & (phi is not None) & (psi is not None):
+        R = 3 / 10 * F_plus_squared(theta, phi, psi)
+    elif (theta is not None) & (phi is not None) & (psi is None):
+        psi = np.random.uniform(0, 2 * np.pi)
+        R = 3 / 10 * F_plus_squared(theta, phi, psi)
+
     # work out the confusion noise or just set to 0
     if include_confusion_noise:
         cn = Sc(f, t_obs)
     else:
         cn = np.zeros(len(f)) if isinstance(f, (list, np.ndarray)) else 0.0
 
     # calculate sensitivity curve
-    psd = (1 / (L**2) * (Poms(f) + 4 * Pacc(f) / (2 * np.pi * f)**4)) / R + cn
+    psd = (1 / (L ** 2) * (Poms(f) + 4 * Pacc(f) / (2 * np.pi * f) ** 4)) / R + cn
```

Comment:
I usually leave these power signs without the spaces to better associate them with the variable (and distinguish from multiplication). Not a hugely strong opinion though if you prefer them this way.